.github/CODEOWNERS

```diff
@@ -281,6 +281,10 @@
 # ServiceLabel: %Digital Twins %Service Attention
 /sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @sourabhguha @inesk-vt @jamdavi
```

Comment:
I updated this last week to remove sourabh and ines. Are you fully rebased on master?

Reply:
I haven't rebased on master. I'll take a look at the diffs in ADT between when I branched off of master and today. Thanks for letting me know.